app/browser/reducers/windowsReducer.js

```diff
@@ -332,6 +333,7 @@ const windowsReducer = (state, action, immutableAction) => {
       break
     case appConstants.APP_WINDOW_CREATED:
       state = windowState.maybeCreateWindow(state, action)
+      windowActions.onWindowFocus(action.getIn(['windowValue', 'windowId']), action.get('windowValue'))
```

Comment:
I don't think we can assume that app window created means always focused.  You could maybe check the window here and if focused do that if for some reason the `focus` event isn't firing.

Reply:
done